src/control/server/ctl_harness.go

```diff
@@ -99,16 +105,19 @@ func (hc *harnessClient) processReturn(rpcResp *mgmtpb.RanksResp) (system.Member
 //
 // Results are returned for the ranks specified in the input parameter
 // if they are being managed by the remote control server.
-func (hc *harnessClient) Query(ctx context.Context, addr string, ranks ...uint32) (system.MemberResults, error) {
+func (hc *harnessClient) Query(parent context.Context, addr string, ranks ...uint32) (system.MemberResults, error) {
 	hc.log.Debugf("issuing Query request to harness at %s", addr)
 
 	rpcReq, err := hc.prepareRequest(RemoteHarnessReq{
-		Addr: addr, Ranks: ranks, Force: false, Timeout: float32(defaultTimeout),
+		Addr: addr, Ranks: ranks, Force: false, Timeout: int64(harnessClientRankTimeount),
```

Comment:
Instead of converting these, you could use the .Nanoseconds() method which returns an int64 representation of the Duration. In the end it's the same result, but might be a little less error-prone down the road to get exactly what you ask for rather than assuming that the type conversion will do what you expect.